model/godleyTableWindow.cc

```diff
@@ -483,15 +483,17 @@ namespace minsky
     motionRow=motionCol=-1;
     if (selectedRow==0)
       {
-        if (c>0 && c!=selectedCol)
+        if (c>0 && selectedCol!=0 && c!=selectedCol)      // Cannot move Intitial Conditions cell from col 0 to one with a "NoAssetClass" designation. For ticket 1064/1066
```

Comment:
Comment doesn't apply = (0,0) is not the initial conditions cell, but the one labelled "flows V/stocks ->"
Perhaps change the comment to read "disallow moving flow labels column"

Reply:
But nice catch!

Reply:
Thanks. :)  I've replaced the comment with your suggestion.